blockchain/db/postgres/transaction_db.py

```diff
@@ -54,13 +54,14 @@
                             family_of_business,
                             line_of_business,
                             payload,
+                            payload_hash,
```

Comment:
Want to discuss payload hash being in the table separate from the other hashes (full and stripped hashes are in the signature structure, stored as JSON type).

While this makes sense for the use case of querying for the hash of an arbitrary payload, it may suit us to store payload_hash in sig and duplicate it here along with the other hashese.